docs/workstation_release_management.rst

```diff
@@ -315,23 +271,24 @@ You can then proceed with distributing the package, via the “test” or
    %_signature gpg
    %_gpg_name 22245C81E3BAEB4138B36061310F561200F4AD77
 
-Distributing packages
----------------------
-
-For the Debian packages, see
-https://github.com/freedomofpress/securedrop-debian-packaging/. For the
-RPM packages, such as the ``securedrop-workstation`` TemplateVM package,
-first build the package (e.g. ``make template``), then sign the RPM, as
-outlined above.
+.. _How to create the DispVM for building packages:
 
-To upload the package, submit a PR to
-https://github.com/freedomofpress/securedrop-workstation-dev-rpm-packages-lfs/
+How to create the DispVM for building packages
+==============================================
 
-The RPM will immediately be available in dom0. Provided you’ve run the
-Salt configurations, find it via:
-
-::
+To avoid inadvertently contaminating a build environment with development changes, we'll use a DispVM for building SecureDrop Workstation packages. To do this, we'll create a VM hierarchy with a Debian 11 TemplateVM (for customizing system packages), an AppVM based on that TemplateVM (to customize home directory), and finally a DispVM that reuses that AppVM image and deletes customizations on each run.
```

Comment:
This is super clear, I love it!

Reply:
I agree—and so appreciate this revision, @creviera.  My only thought is that this risks duplicating (e.g.) <https://github.com/freedomofpress/securedrop-builder/blob/7cd22476706b1c25f6536d52d27ab134b415664a/README.md?plain=1#L112-L129>.  Can these requirements and steps live someplace canonical?

Reply:
You're right! I've got to make up my mind an choose where these instructions should live because both draft PRs add them in separate places. Any preference? I think here makes more sense since it's more of a requirement of our release process rather than a build requirement.

Reply:
Ok, I left a TODO note in the other PR to link here.